spring-cloud-gcp-secretmanager/src/test/java/com/google/cloud/spring/secretmanager/it/SecretManagerTestConfiguration.java

```diff
@@ -92,7 +92,7 @@ public SecretManagerServiceClient secretManagerClient() throws IOException {
 
   @Bean
   public SecretManagerTemplate secretManagerTemplate(SecretManagerServiceClient client) {
-    return new SecretManagerTemplate(client, this.projectIdProvider);
+    return new SecretManagerTemplate(client, this.projectIdProvider, false);
```

Comment:
(mostly non-actionable philosophizing) This is a good example of how providing backwards compatibility, in addition to being good for the users, is also good for us. If a backwards-compatible constructor or setter modifying default behavior were available, then this file would not need to change at all.

Reply:
Thanks for the advice, now this modification is removed from this PR.